libsrc/geos-apple/targetutil/Makefile.inc

```diff
@@ -5,8 +5,7 @@ DEPS += ../libwrk/$(TARGET)/convert.d
 ../libwrk/$(TARGET)/convert.o: $(SRCDIR)/targetutil/convert.c | ../libwrk/$(TARGET)
 	$(COMPILE_recipe)
 
-../lib/apple2enh.lib:
-	@$(MAKE) --no-print-directory apple2enh
+../lib/apple2enh.lib: apple2enh
```

Comment:
I'm afraid that this isn't a fix :-(

See https://github.com/cc65/cc65/blob/master/libsrc/Makefile#L66-L73

I guess your approach builds an apple2enh.lib with some GEOS files...

Reply:
*Argh*
I see.

I was too enthusiastic, then.

As I do not know anything about the Apple II, I could not even test it.

As I said, I do not understand your build system enough.

I am sorry. So I am retracting my PR.

Reply:
Apart from that it doesn't seem to work for me in the first place:

```
make geos-apple
../bin/ld65 -o ../target/geos-apple/util/convert.system -C apple2enh-system.cfg ../libwrk/geos-apple/convert.o ../lib/apple2enh.lib
ld65: Error: Input file '../lib/apple2enh.lib' not found
```

Reply:
That's interesting. I just re-tried, the compilation did not fail. It succeeded (but, most probably, the output is wrong, as you stated).

Reply:
> I am sorry.

Don't! Thanks for
* The successful problem analysis
* The effort spent to provide a fix